crates/payload/builder/src/traits.rs

```diff
@@ -25,7 +25,7 @@ pub trait PayloadJob: Future<Output = Result<(), PayloadBuilderError>> + Send +
         + Sync
         + 'static;
     /// Represents the built payload type that is returned to the CL.
-    type BuiltPayload: BuiltPayload + std::fmt::Debug;
+    type BuiltPayload: BuiltPayload + Clone + std::fmt::Debug;
```

Comment:
I think this is reasonable bound, shouldn't impact anything.

and I don't think the clone is that expensive